packages/node/src/node.ts

```diff
@@ -51,17 +49,15 @@ export class Node {
       this.signer.address,
       networkContext,
       this.storeService,
-      // account-address-based indexing
       `${nodeConfig.STORE_KEY_PREFIX}/${this.signer.address}`
     );
-    this.instructionExecutor = new InstructionExecutor(networkContext);
     this.registerMessagingConnection();
     this.requestHandler = new RequestHandler(
       this.incoming,
       this.outgoing,
       this.channels,
       this.messagingService,
-      this.instructionExecutor
+      new InstructionExecutor(networkContext)
```

Comment:
As context, my initial motivation for having it be a property on `Node` was that I had intended for middlewares to be registered at this scope rather than inside the `RequestHandler`. NBD, makes sense to do this for now.